requirements-links.txt

```diff
@@ -1 +0,0 @@
-git+https://github.com/eclipse-velocitas/vehicle-model-python.git@v0.3.0
```

Comment:
This version is ancient - I assume it is not needed, or?

Reply:
It's still needed at the moment, I think. But yes, would like to get rid of it ...

Reply:
When/how is it needed/used?